es6-classes/circle.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super(radius);
+    this.radius = radius;
+    this.area = (radius * radius) * 3.14;
+    this.perimeter = 2 * 3.14 * radius;
```

Comment:
Instead of setting these properties manually, we should take advantage of the `super` keyword. This allows you to call methods on the parent class.